### PR TITLE
S2U-24 Update I18N Strings

### PR DESCRIPTION
**Pull request by stetsche**
https://sakaiproject.atlassian.net/browse/S2U-24

### CODE REVIEW

**Comment by stetsche** [comment]
Thanks @NuriaQS and @DavidVegaSa for ES and CA translations!

**Comment by stetsche** [comment]
Hi @PabloSanRoman could you please help with the following translations?
DataTablesMessages.properties:
```
infoEmpty=No {0} found
infoFiltered=(filtered from _MAX_ total {0})
lengthMenu=Display _MENU_ {0} per page
entities=results
```
EventLogMessages.properties:
```
export_csv=Export as CSV
datatables_entities=events
```
Here screenshots for context:
image
image

**Comment by mpellicer** [comment]
Thanks, we need files for _eu

**Review by mpellicer — CHANGES_REQUESTED** [review]
Translation keys for _eu requested

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EventLogMessages_eu.properties [hunk 1]
```diff
@@ -59,3 +59,8 @@ alt_sortErrorMsgDescending=eherantz ordenatu erroreekin
 t_sortIP=IP helbideekin ordenatu
 alt_sortIPAscending=Gorantz ordenatu IP helbideekin
 alt_sortIPDescending=Beherantz ordenatu IP helbideekin
+
+# S2U-24
```
Comment (mpellicer) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EventLogMessages_eu.properties:63]: @pablosanroman Would you mind translating these strings?

**Comment by mpellicer** [comment]
This requires rebasing, sorry @stetsche, I needed to reword some of the commits because they did not contain the contract number.

**Comment by stetsche** [comment]
@mpellicer No problem, done!

**Review by mpellicer — APPROVED** [review]

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EventLogMessages_eu.properties (hunk 1 above)
Comment (PabloSanRoman) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EventLogMessages_eu.properties:63]: Hi @mpellicer 
export_csv=Esportatu CSV
datatables_entities=gertaerak

**Comment by PabloSanRoman** [comment]
Hi  @stetsche, very useful the image-captions for context, thank you! 
(note: "showing page.." text, we can see on the caption, doesn't exist on these properties. 

# DataTablesMessages.properties:

infoEmpty=Ez da aurkitu: "{0}"
infoFiltered=(hemendik iragazita: _MAX_ {0} guztiak)
lengthMenu=Erakutsi _MENU_ {0} orrialde bakoitzean
entities=emaitzak

# EventLogMessages.properties:

export_csv=Esportatu CSV
datatables_entities=gertaerak

**Comment by mpellicer** [comment]
> Hi @stetsche, very useful the image-captions for context, thank you! (note: "showing page.." text, we can see on the caption, doesn't exist on these properties.
> 
> # DataTablesMessages.properties:
> infoEmpty=Ez da aurkitu: "{0}" infoFiltered=(hemendik iragazita: _MAX_ {0} guztiak) lengthMenu=Erakutsi _MENU_ {0} orrialde bakoitzean entities=emaitzak
> 
> # EventLogMessages.properties:
> export_csv=Esportatu CSV datatables_entities=gertaerak

Thanks Pablo, strings updated

**Pull request merged**